openstudio-common-measures.gemspec

```diff
@@ -5,26 +5,35 @@ require 'openstudio/common_measures/version'
 Gem::Specification.new do |spec|
   spec.name          = 'openstudio-common-measures'
   spec.version       = OpenStudio::CommonMeasures::VERSION
+  spec.platform      = Gem::Platform::RUBY
   spec.authors       = ['David Goldwasser', 'Nicholas Long']
   spec.email         = ['david.goldwasser@nrel.gov', 'nicholas.long@nrel.gov']
 
+  spec.homepage      = 'https://openstudio.net'
   spec.summary       = 'Common library and measures for OpenStudio'
   spec.description   = 'Common library and measures for OpenStudio'
-  spec.homepage      = 'https://openstudio.net'
+  spec.metadata = {
+      'bug_tracker_uri' => 'https://github.com/NREL/openstudio-common-measures-gem/issues',
+      'changelog_uri' => 'https://github.com/NREL/openstudio-common-measures-gem/blob/develop/CHANGELOG.md',
+      #'documentation_uri' =>  'https://www.rubydoc.info/gems/openstudio-common-measures-gem/#{gem.version}',
+      'source_code_uri' => "https://github.com/NREL/openstudio-common-measures-gem/tree/v#{spec.version}"
+  }
 
-  # Specify which files should be added to the gem when it is released.
-  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files         = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
   end
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir = 'exe'
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'openstudio-extension', '~> 0.1.6'
-  spec.add_dependency 'openstudio-standards', '~> 0.2.10'
+  spec.required_ruby_version = '~> 2.5.0'
+
+  spec.add_dependency 'bundler', '~> 2.1'
+  spec.add_dependency 'openstudio-extension', '~> 0.2.0'
+  spec.add_dependency 'parallel', '~> 1.12.0'
 
-  spec.add_development_dependency 'bundler', '~> 1.14'
-  spec.add_development_dependency 'rake', '12.3.1'
-  spec.add_development_dependency 'rspec', '3.7.0'
+  spec.add_development_dependency 'github_api', '~> 0.18.0'
+  spec.add_development_dependency 'rake', '~> 12.3'
+  spec.add_development_dependency 'rspec', '~> 3.7'
+  spec.add_development_dependency 'rubocop', '~> 0.54.0'
```

Comment:
```
   spec.add_dependency 'parallel', '~> 1.19.1' 

   spec.add_development_dependency 'rake', '~> 13.0.1'
   spec.add_development_dependency 'rspec', '~> 3.9'
   spec.add_development_dependency 'rubocop', '~> 0.80.0'
```

Reply:
I think it would be great to update all the dependencies to the latest. Don't we want to start here: https://github.com/NREL/openstudio-gems/blob/develop/openstudio-gems.gemspec and here: https://github.com/NREL/openstudio-gems/blob/develop/Gemfile

Reply:
`'rubocop', '~> 0.80.0'` require jaro_winkler, which is a native extension... so that will definitely need to be added to openstudio-gems.